### PR TITLE
Pass right args to _updateKnob

### LINKED ISSUE #131: Accessibility: Invalid aria properties after load
In `ready`, `_updateKnob` is called with only one value. This results in setting the other aria attributes to `undefined`. The end result is that this:

```
<paper-slider value="4" min="0" max="10" snaps step="1"></paper-slider>
```

becomes this on load:

```
<paper-slider value="4" min="0" max="10" snaps="" step="1"
    class="style-scope x-foo x-scope paper-slider-0" role="slider" tabindex="0"
    aria-disabled="false" aria-valuemin="undefined" aria-valuemax="undefined" aria-valuenow="4">
```

which fails [AX_ARIA_04](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_04).

Why do we call `_updateKnob` with just one value?

http://jsbin.com/dorezelodo/

### PR DESCRIPTION
**Pull request by keanulee**
Fixes #131 

cc @mgiuffrida

### CODE REVIEW

File: paper-slider.html [hunk 1]
```diff
@@ -474,7 +474,7 @@
       ready: function() {
         // issue polymer/polymer#1305
         this.async(function() {
-          this._updateKnob(this.value);
+          this._updateKnob(this.value, this.min, this.max);
         }, 1);
```
Comment (frankiefu) [inline paper-slider.html]: I don't think we need to call `_updateKnob` here at all inside `ready`.  the [observer](https://github.com/PolymerElements/paper-slider/blob/01ad379495af47f8b77e1361fe014415001d3dff/paper-slider.html#L458) should be already taking care of calling `_updateKnob` at the right time with the right values.

Comment (keanulee) [inline paper-slider.html]: I didn't really read the above mentioned issue (polymer/polymer#1305) - not sure if it's still relevant.

Comment (frankiefu) [inline paper-slider.html]: Afaict, the comment refers to timing issue when using `async with 0 (or no time) and 1`, not if the `_updateKnob` call is actually needed at all.

Would you mind removing the call and see if things (unit test and demo test) still function correctly?

Comment (mgiuffrida) [inline paper-slider.html]: Yeah it's odd that it'st async. Presumably there was originally a reason for that...

I'll test it out next week when I'm physically back at my machine in SFO.

Comment (keanulee) [inline paper-slider.html]: @frankiefu demo looks good and tests still pass. Should I remove it (and hopefully nothing breaks)?

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**